src/test/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperationTest.java

```diff
@@ -202,36 +206,48 @@ void shouldValidate_when_organisation_has_correct_roles_complex() throws Excepti
 
     @Test
     void should_fail_validate_when_organisation_has_null_role() throws Exception {
```

Comment:
I think if all these edited tests in this class have now been moved over to the validator test one (which I think they have?) then no longer need them in here anymore? 😃